### PR TITLE
Add suburb, lat/lon to pg_address

### PR DESCRIPTION
**Pull request by vpikulik**

### CODE REVIEW

File: pg_extensions/pg_address/tests/smoke.sql [hunk 1]
```diff
@@ -22,17 +22,20 @@ $$ LANGUAGE SQL;
 
 BEGIN;
 
-SELECT plan(7);
+SELECT plan(10);
 
 SELECT set_up();
 INSERT INTO test_address (address) VALUES
-    (ROW('Germany', 'Berlinregio', 'Berlin', '10117', 'Haubtstr.', '57b'));
+    (ROW('Germany', 'Berlinregio', 'Berlin', '10117', 'Haubtstr.', '57b', 'Mitte', 51.2, 4.32));
```
Comment (pbrinkmannlot) [inline pg_extensions/pg_address/tests/smoke.sql]: Hauptstr. with "p" not with "b" :-)

**Comment by pbrinkmannlot** [comment]
:+1:

File: pg_extensions/pg_address/tests/smoke.sql (hunk 1 above)
Comment (vpikulik) [reply pg_extensions/pg_address/tests/smoke.sql]: I will change this next time )

**Comment by AnnaTishchenko** [comment]
:+1:

**Comment by kuza** [comment]
👍

**Pull request merged**